tools/debian/control

```diff
@@ -92,15 +89,6 @@ Description: Cockpit user interface for networking
  The Cockpit components for interacting with networking configuration.
  Incompatible with systemd-networkd/ifupdown defined networks.
 
-Package: cockpit-pcp
```

Comment:
Please add `Conflicts:/Replaces/Provides: cockpit-pcp` to cockpit-bridge.

https://www.debian.org/doc/debian-policy/ch-relationships.html#replacing-whole-packages-forcing-their-removal